ubuntu/content.md

```diff
@@ -16,6 +16,18 @@ The `ubuntu:latest` tag points to the "latest LTS", since that's the version rec
 
 Along a similar vein, the `ubuntu:devel` tag is an alias for whichever release the "devel" suite on the mirrors currently points to, as determined by the following one-liner: `wget -qO- http://archive.ubuntu.com/ubuntu/dists/devel/Release | awk -F ': ' '$1 == "Codename" { print $2; exit }'`
 
+## Locales
+
+Given that it is a minimal install of Ubuntu, this image only includes the `C`, `C.UTF-8`, and `POSIX` locales by default. For most uses requiring a UTF-8 locale, `C.UTF-8` is likely sufficient (`-e LANG=C.UTF-8` or `ENV LANG C.UTF-8`).
```

Comment:
`Given that it is a minimal install`

if this true, why does it come installed with 100+ packages including `libstdc++6`? stripping off other optional packages will justify this statement and lack of locales. it will also make it a true "minimal" image like alpine; no c++ library is installed by default